### ISSUE TITLE
SPMC 16.7.0 issue with accessing Android apps from Favorite folder

### ISSUE THREAD

**Issue by redan21** [koying/SPMC#933]
I had to downgrade back to 16.6.0 because I wasn't able to utilize the favorites folder to make the most use out of my Fire Stick. It will load into the folder just fine but when trying to click the app it will bring me back to the main screen

**Comment by TheseUserNamesAreMakingMeThirsty** [comment]
Covered already - [SPMC 16.7.0 #911](https://github.com/koying/SPMC/issues/911)  & [SPMC 16.7 Open with issues and Plex #917](https://github.com/koying/SPMC/issues/917)

**Comment by redan21** [comment]
Covered but not yet fixed :) or has it? As of 2 nights ago I was still
havning that issue

**Comment by TheseUserNamesAreMakingMeThirsty** [comment]
No, not fixed, an update is required for that.
At least you could downgrade, I will have to do a uninstall and reinstall 16.6.0 as it wont let me roll back.

**Comment by redan21** [comment]
I didn't know I could downgrade either lol I had to do the same thing as I
didn't bother to backup any data
:( so annoying but worth it once all my widgets and stuffs are back :) if I
can be of any assistance let me know!

**Comment by redan21** [comment]
So will you be releasing a new version with that and whatever other issues
fixed, or will it just be slightly a bug-fix update but still 16.7.0?

**Comment by redan21** [comment]
Oh wait you're not the developer, my mistake. Maybe you have some insider info on what koying is planning though?

**Comment by TheseUserNamesAreMakingMeThirsty** [comment]
Yeah, not the dev. I've been on GitHub for 5 days so couldn't be further from it. 
No it won't be 16.7.0 it will have to change to reflect a change in the code, so something like 16.7.1.
